project/gen2.scala

```diff
@@ -32,13 +32,17 @@ object gen2 {
   implicit val AngleMeta: Meta[Angle] =
     Meta[Double].xmap(Angle.fromArcsecs, _.toArcsecs)
 
+  implicit val ZoneIdMeta: Meta[ZoneId] =
+    Meta[String].xmap(ZoneId.of, _.toString)
```

Comment:
`ZoneId.of` can throw an exception for unknown zone ids.Would that affect the code reading from the db?